.github/workflows/trunk.yml

```
@@ -96,7 +96,7 @@ jobs:
         run: yarn ${{ matrix.script }}
 
       - name: Upload an integration test artifact
-        if: ${{ matrix.script == 'integration:ng15' }}
+        if: ${{ matrix.script == 'integration:ng16' }}
```

This should run for all integration tests.
```suggestion
        if: ${{ startsWith( matrix.script, 'integration:' ) }}
```